networking/eth2/src/test/java/tech/pegasys/teku/networking/eth2/gossip/topics/AggregateTopicHandlerTest.java

```diff
@@ -61,11 +57,27 @@ public void handleMessage_validAggregate() {
     assertThat(result).isCompletedWithValue(ValidationResult.Valid);
   }
 
+  @Test
+  public void handleMessage_invalidAggregate_wrongFork() {
+    final ValidateableAttestation aggregate =
+        ValidateableAttestation.aggregateFromValidator(
+            spec, dataStructureUtil.randomSignedAggregateAndProof(wrongForkSlot));
+    when(processor.process(aggregate))
+        .thenReturn(SafeFuture.completedFuture(InternalValidationResult.ACCEPT));
```

Comment:
This is a little misleading, because the milestone validator is now whats causing things to be invalid...
I think i'd prefer to have no when here, and at the end of the test
`verifyNoInteractions(processor);`